saleor/plugins/webhook/tests/test_webhook.py

```diff
@@ -618,6 +619,39 @@ def test_checkout_updated(
     )
 
 
+@freeze_time("1914-06-28 10:50")
+@mock.patch("saleor.checkout.calculations.fetch_checkout_prices_if_expired")
+@mock.patch("saleor.plugins.webhook.plugin._get_webhooks_for_event")
+@mock.patch("saleor.plugins.webhook.plugin.trigger_webhook_sync")
+def test_get_shipping_methods_for_checkout_untaxed(
```

Comment:
What exacly this test is testing? I see that the assert at the end is that we expect that we don't make any call, but the name of the test doesn't say that this is expected.

Reply:
I've changed the name of this test function, it should be clearer now what is it testing